certified-connectors/AXtension Content Gate/apiDefinition.swagger.json

```diff
@@ -15,655 +15,370 @@
+                  "id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "id"
+                  },
+                  "contentEntityTemplateId": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "contentEntityTemplateId"
+                  },
+                  "businessEntityId": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "businessEntityId"
+                  },
+                  "status": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "status"
+                  },
+                  "createdOn": {
+                    "type": "string",
+                    "description": "createdOn"
+                  },
+                  "createdBy": {
+                    "type": "string",
+                    "description": "createdBy"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "List content item requirements",
+        "description": "Lists the content item requirements",
+        "operationId": "List-Content-Entity-Requirements",
+        "parameters": [
+          {
+            "name": "providerReferenceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The business entity connector.",
+            "x-ms-summary": "Business entity connector",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "List-Business-Entity-Connectors",
+              "value-path": "referenceId",
+              "value-title": "name"
+            }
+          },
+          {
+            "name": "externalType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The business entity type.",
+            "x-ms-summary": "Business entity type",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "List-Business-Entity-Provider-Types",
+              "value-path": "provider/externalType",
+              "value-title": "name",
+              "parameters": {
+                "providerReferenceId": {
+                  "parameter": "providerReferenceId"
+                }
+              }
+            }
+          },
+          {
+            "name": "externalId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The business entity identifier.",
+            "x-ms-summary": "Business entity identifier",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```

Comment:
@JoepKillaars Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.